ana_scans.py

```diff
@@ -123,9 +124,9 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   #Execute Commands
   try:
     log = file("%s/anaLog.log"%(dirPath),"w")
-   
-    runCommand(cmd,log)
-    #runCommand(cmd)
+ 
+    #runCommand(cmd,log)
+    runCommand(cmd)
     for key in postCmds:
       runCommand(postCmds[key])
```

Comment:
This is not true of python. postCmds can still be a list and you can append lists into it. Then when you iterate over postCmds you will get the lists which define the commands to be run. The difference between a list and a dict is that a dict uses a map to associate a key to the location of some data. Dicts are built on top of the list so you can access entries with keys instead of just having to pass an int which gets the object placed with the Nth call to append on the list object.